other/upgrade_2-1_postgresql.sql

```diff
@@ -1490,6 +1574,13 @@ ADD COLUMN likes smallint NOT NULL default '0';
 /******************************************************************************/
 --- Adding support for mentions
 /******************************************************************************/
+---# upgrade check
+---{
+$table_columns = $smcFunc['db_list_columns']('{db_prefix}mentions');
```

Comment:
This will fail and cause the upgrader to halt if the table doesn't exist. Instead, after the `CREATE TABLE`, maybe we should test whether the indexes already exist or not.

Reply:
Like i said,
the upgrader is tested on pg.
so db_list_columns works nice here to test the existing of table